src/utils/mellowHelpers/geckoEthToUsd.ts

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable no-await-in-loop */
+import axios from 'axios';
+import { sentryTracker } from '../sentry';
+
+export async function geckoEthToUsd(): Promise<number> {
+  for (let attempt = 0; attempt < 5; attempt += 1) {
+    try {
+      const data = await axios.get(
+        `https://pro-api.coingecko.com/api/v3/simple/price?x_cg_pro_api_key=${process.env.REACT_APP_COINGECKO_API_KEY}&ids=ethereum&vs_currencies=usd`,
```

Comment:
This pattern is being used in amm.ts and borrowAMM.ts as well. The only instance that uses api key as a param is in community sbt. There already exist many instances of duplication and this should be part of a refactoring ticket. Unless there is a technical reason which would prevent this implementation from working then I would suggest to use this method. Let's discuss this on a call today as well.

Reply:
I see. So Community SBT is doing something different then. Since I worked with Ioana during that time, this is what I thought as well. Let's create a notion ticket on the matter of refactor?

Reply:
Created the ticket in engineering BAU